### PR TITLE
RHTAPWATCH-666: Deploy smee server for prod

### PR DESCRIPTION
**Pull request by gbenhaim**
Deploy the smee server to the production host cluster. It will be used for our internal production environment.

### CODE REVIEW

File: components/smee/production/ip-allow-list.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (ifireball) [inline components/smee/production/ip-allow-list.yaml:1]: We should copy the comment about where the addresses came from here.

**Review by ifireball — CHANGES_REQUESTED** [review]

File: components/smee/production/ip-allow-list.yaml (hunk 1 above)
Comment (gbenhaim) [reply components/smee/production/ip-allow-list.yaml:1]: done

**Review by ifireball — APPROVED** [review]
Lgtm

**Comment by hugares** [comment]
@gbenhaim You need to rebase this one to get test passing. You are getting an issue that was fixed yesterday.

**Pull request merged**